.github/workflows/ci.yml

```diff
@@ -50,22 +66,21 @@ jobs:
       - name: Build binary
         run: sbt --client examples/nativeLink
 
-      - name: Push cache 
+      - name: Push cache
         run: sbt --client pushRemoteCache || echo 'oh well then'
 
       - name: Publish ${{ github.ref }}
         if: github.event_name != 'pull_request' && (startsWith(github.ref, 'refs/tags/v') || github.ref == 'refs/heads/main')
-        run: sbt ci-release
+        run: sbt --client ci-release
```

Comment:
Actually this is wrong – `sbt --client` won't pick up the `env` variables from the block below (SBT server will only see the env variables available on startup), so the secrets and passphrases won't be propagated.

So this needs to go back to being a `sbt ci-release`

Reply:
Oh, interesting, I didn't know that. Fixed.